### PR TITLE
Update CCAnimation3D.cpp

### PR DESCRIPTION
**Pull request by hcqmaker**
FileUtils use lookup file this will be fail

### CODE REVIEW

**Comment by hcqmaker** [comment]
FileUtils use lookup file this will be fail

**Comment by minggo** [comment]
@super626 
Could you please take a look?

**Comment by dabingnn** [comment]
@hcqmaker Could you please give a more detailed description of the bug? A test case is preferred if you can provide one. Thanks.

File: cocos/3d/CCAnimation3D.cpp [hunk 1]
```diff
@@ -30,7 +30,8 @@ NS_CC_BEGIN
 
 Animation3D* Animation3D::create(const std::string& fileName, const std::string& animationName)
 {
-    std::string fullPath = FileUtils::getInstance()->fullPathForFilename(fileName);
+    std::string fullPath = fileName;
```
Comment (dabingnn) [inline cocos/3d/CCAnimation3D.cpp:33]: It seems this code will fail if a relative path is used.

**Comment by super626** [comment]
@hcqmaker  
Agree with @dabingnn 
Can you describe it in detail?

**Comment by dabingnn** [comment]
This pr will be closed, if you have any feedbacks, please add a detailed description of bug and reopen it.
Thanks.

**Pull request closed without merging**